src/Main.java

```diff
@@ -1,5 +1,79 @@
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Задание 1,2");
+        int [] weight = new int[]{1, 2, 3};
```

Comment:
здесь нужно было создать массив длины 3 и заполнить его после, потому что текущий способ создания равносилен такому
```suggestion
        int [] weight = {1, 2, 3};
```